### DISCUSSION TITLE
Customizable Slack Deployment Notifications

### DISCUSSION THREAD

**Discussion by AngellusMortis** [community/community#13124] [Actions]
A tangent from an issue I made about [deployment review request notifications](https://github.com/integrations/slack/issues/1364) not working, it would be great to be able to customize these notifications.

As a result of the bug, we are making our own notifications for deployments and we are specifically adding the Pull Request for the code (even if the PR is merged, viewing the PR is generally preferable to viewing the raw commit) and an reference issue in our issue tracking system (we do not use Github Issues/Projects at the moment).

**Comment by ethomson** [comment]
Thanks for this feedback as well, @AngellusMortis - I'll pass this along to the team also.
